core/templates/pages/exploration-player-page/services/current-interaction.service.ts

```diff
@@ -52,11 +75,11 @@ export class CurrentInteractionService {
   constructor(
     private contextService: ContextService,
     private playerPositionService: PlayerPositionService,
-    private playerTranscriptService: PlayerTranscriptService) {}
+    private playerTranscriptService: PlayerTranscriptService) { }
 
-  private static submitAnswerFn: SubmitAnswerFn = null;
-  private static onSubmitFn: OnSubmitFn = null;
-  private static validityCheckFn: ValidityCheckFn = null;
+  private static submitAnswerFn: SubmitAnswerFn | null = null;
+  private static onSubmitFn: OnSubmitFn | null = null;
+  private static validityCheckFn: ValidityCheckFn | null = null;
```

Comment:
Why `null` can be in these?

Reply:
Removed `null` for `onSubmitFn` , for the other two properties it is later explained in the file
https://github.com/oppia/oppia/blob/741cc401ec375d4bd148dd547a718db7bcdb7499/core/templates/pages/exploration-player-page/services/current-interaction.service.ts#L79-L87

Reply:
Please add a comment here too

Reply:
done